gradle.properties

```diff
@@ -0,0 +1,20 @@
+# Project-wide Gradle settings.
+
+# For more details on how to configure your build environment visit
+# http://www.gradle.org/docs/current/userguide/build_environment.html
+
+# Specifies the JVM arguments used for the daemon process.
+# The setting is particularly useful for tweaking memory settings.
+# Default value: -Xmx10248m -XX:MaxPermSize=256m
+# org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError
+
+org.gradle.parallel=true
+org.gradle.daemon=true
+org.gradle.caching=true
+org.gradle.jvmargs=-Dfile.encoding=UTF-8
+kotlin.code.style=official
+
+jabelVersion=1.0.2-PKWARE-SNAPSHOT
```

Comment:
Let's go with `1.0.1-1` - it's version 1.0.1 of the upstream code, and it's our first release of that "code snapshot"